src/main/resources/templates/application/proceedings-scope-limitations-confirm.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en"
+      th:replace="~{layout :: layout(title=~{::title},mainContent=~{::#main-content},pageCategory=${'cases'},breadcrumbs=~{::#breadcrumbs})}"
+      xmlns:th="http://www.thymeleaf.org" xmlns="http://www.w3.org/1999/html">
+<head>
+  <title>LAA CCMS - Create application</title>
+</head>
+<body>
+<div class="govuk-width-container">
+  <div id="breadcrumbs">
+    <a th:href="@{/application/proceedings/{action}/confirm(action=${proceedingFlow.action})}" class="govuk-back-link">Cancel and return to proceeding details</a>
+  </div>
+  <div id="main-content">
+    <form th:action="@{/application/proceedings/scope-limitations/confirm}" method="post">
+      <span class="govuk-caption-l">Add scope limitation</span>
+      <h1 class="govuk-heading-l">
+        Scope limitation details
+      </h1>
+      <p class="govuk-body-l">
+        Please review the scope limitation wording and click 'confirm' to continue.
```

Comment:
Guess it should be 'Confirm' to match button text.

Reply:
amended